net/redis/files/redis.init

```diff
@@ -0,0 +1,39 @@
+#!/bin/sh /etc/rc.common
+
+START=98
+STOP=81
+
+SERVICE_USE_PID=1
```

Comment:
@artynet Here is documentation on writing procd init scripts: https://wiki.openwrt.org/inbox/procd-init-scripts
`redis-server` already seems to run in the foreground, so it should be easy to adapt to procd.